kerlescan/inventory_service_interface.py

```diff
@@ -22,7 +23,9 @@ def ensure_correct_system_count(system_ids_requested, result):
         )
 
 
-def interleave_systems_and_profiles(systems_result, system_profiles_result):
+def interleave_systems_and_profiles(
+    systems_result, system_profiles_result, system_tags_result
+):
```

Comment:
would it be useful if inventory had a "give me everything" API? so you don't have to call it 3x

Reply:
Yes, it definitely would.  Maybe tags should/could be in the metadata, although we will want them in the system profile.  I still have to look at what needs to be done for HSP.  Do we write the historical profiles?  I also need to look at system-baseline-backend as it calls parse_profile in one place (views/v1.py)

Reply:
BTW - I saw a discussion yesterday surrounding the all_installed_packages enhancement request that maybe the inventory API should provide a way for the caller to request what data they want in the response.